### DISCUSSION TITLE
关于地图设计问题

### DISCUSSION THREAD

**Discussion by huwenjing0819** [liusy58/NJU_AdvancedProgramming_SP22#173] [Q&A] [answered]
请问类《植物大战僵尸》的路径设计和类《国王保卫战》的路径设计在整个proj2中是否都可行？（就是说是否一定要设计拐弯）如果都可行的话，在最终的评分环节会有分数上的差异吗？说明文档中好像没有明确的规定🤔

**Comment by liusy58** [comment, marked as answer]
确实都可行，但是尽量做的丰富一些吧，这一块的基础分数不会有很大的差异。

**Reply by liusy58** [reply]
anyway, learning something is the most important. ;-)
